lndb/__main__.py

```diff
@@ -87,6 +88,18 @@
 aa = migr.add_argument
 aa("action", choices=["generate"], help="Generate migration.")
 
+# track anotebook (init nbproject metadata)
+track_parser = subparsers.add_parser("track", help=track_help)
+aa = track_parser.add_argument
+nb_path_help = "A path to the notebook to track."
+aa("nb_path", type=str, metavar="filepath", help=nb_path_help)
+parent_help = (
+    "One or more (delimited by ',') ids of direct ancestors in a notebook pipeline"
+)
+aa("--parent", type=str, metavar="parent", default=None, help=parent_help)
```

Comment:
I'd not support the parent argument as this is something that shouldn't be needed in the context of lamindb!